cmd/okctl/venv.go

```diff
@@ -56,6 +59,35 @@ func buildVenvCommand(o *okctl.Okctl) *cobra.Command {
 			return nil
 		},
 		RunE: func(_ *cobra.Command, args []string) error {
+			handlers := o.StateHandlers(o.StateNodes())
```

Comment:
I compared, and it seems venv.go and show.go contains 100% the same code for this kubeconfig part. Is there any issue with de-duplicating this logic?

Reply:
We should probably move into a common function, but I think we are ok with that duplication for now